src/test/java/org/folio/dew/batch/bursarfeesfines/service/BursarTokenFormatterTest.java

```diff
@@ -0,0 +1,980 @@
+        itemDataToken,
+        accountWithAncillaryData
+      ),
+      is("test_locid")
+    );
+
+    itemDataToken.setValue(BursarExportTokenItemData.ValueEnum.NAME);
+    assertThat(
+      BursarTokenFormatter.formatItemDataToken(
+        itemDataToken,
+        accountWithAncillaryData
+      ),
+      is("test_title")
+    );
+
+    itemDataToken.setValue(BursarExportTokenItemData.ValueEnum.BARCODE);
+    assertThat(
+      BursarTokenFormatter.formatItemDataToken(
+        itemDataToken,
+        accountWithAncillaryData
+      ),
+      is("test_barcode")
+    );
+
+    itemDataToken.setValue(BursarExportTokenItemData.ValueEnum.MATERIAL_TYPE);
+    assertThat(
+      BursarTokenFormatter.formatItemDataToken(
+        itemDataToken,
+        accountWithAncillaryData
+      ),
+      is("test_matname")
+    );
+
+    // test null
+    AccountWithAncillaryData tempAccountWithAncillaryData = AccountWithAncillaryData
+      .builder()
+      .item(null)
+      .build();
+    itemDataToken.setValue(BursarExportTokenItemData.ValueEnum.NAME);
+    assertThat(
+      BursarTokenFormatter.formatItemDataToken(
+        itemDataToken,
+        tempAccountWithAncillaryData
+      ),
+      is("placeholder")
+    );
+
+    tempAccountWithAncillaryData.setItem(new Item());
+    itemDataToken.setValue(BursarExportTokenItemData.ValueEnum.NAME);
+    assertThat(
+      BursarTokenFormatter.formatItemDataToken(
+        itemDataToken,
+        tempAccountWithAncillaryData
+      ),
+      is("placeholder")
+    );
+  }
+
+  @Test
+  void testFormatConditionalDataToken() {
+    BursarExportTokenConditional conditionalToken = new BursarExportTokenConditional();
+    conditionalToken.setElse(
+      new BursarExportTokenConstant() {
+        {
+          setValue("test_else");
+        }
+      }
+    );
+    // test else
+    assertThat(
+      BursarTokenFormatter.formatConditionalDataToken(
+        conditionalToken,
+        accountWithAncillaryData
+      ),
+      is("test_else")
+    );
+
+    List<BursarExportTokenConditionalConditionsInner> conditions = new ArrayList<>();
+
+    BursarExportTokenConditionalConditionsInner condition1 = new BursarExportTokenConditionalConditionsInner();
+    BursarExportTokenFeeAmount feeAmountToken = new BursarExportTokenFeeAmount();
+    feeAmountToken.setDecimal(true);
+    condition1.setValue(feeAmountToken);
+    BursarExportFilterAmount falseAmountFilter = new BursarExportFilterAmount();
+    falseAmountFilter.setAmount(100);
+    falseAmountFilter.setCondition(ConditionEnum.LESS_THAN);
+    condition1.setCondition(falseAmountFilter);
+    conditions.add(condition1);
+
+    BursarExportTokenConditionalConditionsInner condition2 = new BursarExportTokenConditionalConditionsInner();
+    condition2.setValue(feeAmountToken);
+    condition2.setCondition(new BursarExportFilterPass());
+    conditions.add(condition2);
+    conditionalToken.setConditions(conditions);
+
+    log.info(conditions.toString());
```

Comment:
Should these be here?

Reply:
when these are removed don't forget to remove the `@Log4j2` and corresponding import